### PR TITLE
Singla page web Application

### PR DESCRIPTION
**Pull request by tylher**
- Created a new section for contact information
- Added a footer to the page
- Implemented logic to change content of the page without reloading the page (using JavaScript)
- Added style to each content

### CODE REVIEW

File: awesome_book.js [hunk 1]
```diff
@@ -16,20 +47,23 @@ class BookStore {
...
+      }
+      Book.push(book);
+      const storage = JSON.parse(localStorage.getItem('books'));
+      storage.push(book);
+      const div = document.createElement('div');
+      div.classList.add('align-items');
+      div.innerHTML = `<div class="flex-division-box"><h3>${book.Title}</h3> <p>${book.author}</p></div><button onclick='bookStore.remove(this)' class='remove-book'>remove</button> `;
+      this.displayBooks.appendChild(div);
+      localStorage.setItem('books', JSON.stringify(storage));
+    });
```
Comment (Gambit142) [inline awesome_book.js]: - [ ](Optional) A good user interaction implementation would be resetting the value of the `bookTitle` and `bookAuthor` to an empty string after you add a book. If you want to go a step further, you can display a message underneath the button that shows when a book was successfully added and when it failed to add. 👍

- [x] (Optional) Another good user interaction would be to switch the page to the Booklist page after adding a book 👍

**Review by Gambit142 — APPROVED** [review]
Hi @tylher @Shubh-Dev 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional 
- Eslint helps software developers like us write community standard JavaScript codes. Using `/* eslint-enable */` every time is not a good practice because you ought to investigate the reason why you are having those errors and restructure the code in the right way. Remember that you are not just submitting this repository for approval, but a recruiter might check this repository and see the use of eslint-disable comments throughout the code. 

_**Hint:**_ If you map through an array, you want to return a new array or something. That should fix most of the eslint errors 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: awesome_book.js (hunk 1 above)
Comment (tylher) [reply awesome_book.js]: Really appreciate this, thank you!

**Comment by tylher** [comment]
@Gambit142  Made the optional changes you suggested, thank you ☺️

**Pull request merged**